lib/smart_proxy_openscap/openscap_plugin.rb

```diff
@@ -18,6 +18,7 @@ class Plugin < ::Proxy::Plugin
     https_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
 
     default_settings :spooldir => '/var/spool/foreman-proxy/openscap',
-                     :openscap_send_log_file => '/var/log/foreman-proxy/openscap-send.log'
+                     :openscap_send_log_file => 'logs/openscap-send.log',
+                     :contentdir => 'openscap/content'
```

Comment:
this is missing `/var/spool`, I had to explicitly set the value in config file in order to make it work